### ISSUE TITLE
Create featured favicons later

### ISSUE THREAD

**Issue by rugk** [RealFaviconGenerator/realfavicongenerator#260]
It would be nice to create the featured favicons after creating the favicons itself, e.g. when analysing the favicons or when you provide all the files you generated with RFG.

**Comment by phbernard** [comment]
That's right, the presentation picture creation can bow be postponed. Until a few weeks, this picture, which was slightly different, was displayed in the result page, not in a separated page, so this choice of design made sense. But that's not the case anymore.
